plugins/modules/rds_cluster.py

```diff
@@ -463,6 +472,13 @@
     engine: aurora-postgresql
     state: present
     db_instance_class: 'db.t3.medium'
+
+- name: Remove a cluster from global DB
```

Comment:
What if the user wants to delete the cluster and remove It from the global db cluster at the same time?

Reply:
if deleting a cluster which is part of a global db, then it fails with boto error (cannot delete, part of global db.....)

so with current changes in PR, users can use 1 task to remove cluster from global db and another task to delete the cluster

Reply:
How much does it take to remove the rds cluster from a global one?

Reply:
from my trial-errors, it takes a couple of minutes